{
    "containers": {
        "cna": {
            "affected": [
                {
                    "defaultStatus": "unknown",
                    "product": "Security Guardium",
                    "vendor": "IBM",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "11.1"
                        }
                    ]
                }
            ],
            "datePublished": "2020-06-02T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "IBM Security Guardium 11.1 could allow an attacker on the same network to gain access to the Solr dashboard and cause a denial of service attack. IBM X-Force ID: 176997."
                }
            ],
            "metrics": [
                {
                    "cvssV3_0": {
                        "BM": {
                            "A": "H",
                            "AC": "L",
                            "AV": "A",
                            "C": "N",
                            "I": "N",
                            "PR": "N",
                            "S": "U",
                            "SCORE": "6.500",
                            "UI": "N"
                        },
                        "TM": {
                            "E": "U",
                            "RC": "C",
                            "RL": "O"
                        }
                    }
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "Denial of Service",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2022-01-13T00:00:00",
                "orgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
                "shortName": "ibm"
            },
            "references": [
                {
                    "name": "https://www.ibm.com/support/pages/node/6218974",
                    "url": "https://www.ibm.com/support/pages/node/6218974",
                    "x_refsource": "CONFIRM"
                },
                {
                    "name": "ibm-guardium-cve20204307-dos (176997)",
                    "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/176997",
                    "x_refsource": "XF"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "psirt@us.ibm.com",
                    "DATE_PUBLIC": "2020-06-02T00:00:00",
                    "ID": "CVE-2020-4307",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "Security Guardium",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "11.1"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "IBM"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "IBM Security Guardium 11.1 could allow an attacker on the same network to gain access to the Solr dashboard and cause a denial of service attack. IBM X-Force ID: 176997."
                        }
                    ]
                },
                "impact": {
                    "cvssv3": {
                        "BM": {
                            "A": "H",
                            "AC": "L",
                            "AV": "A",
                            "C": "N",
                            "I": "N",
                            "PR": "N",
                            "S": "U",
                            "SCORE": "6.500",
                            "UI": "N"
                        },
                        "TM": {
                            "E": "U",
                            "RC": "C",
                            "RL": "O"
                        }
                    }
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "Denial of Service"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "https://www.ibm.com/support/pages/node/6218974",
                            "refsource": "CONFIRM",
                            "title": "IBM Security Bulletin 6218974 (Security Guardium)",
                            "url": "https://www.ibm.com/support/pages/node/6218974"
                        },
                        {
                            "name": "ibm-guardium-cve20204307-dos (176997)",
                            "refsource": "XF",
                            "title": "X-Force Vulnerability Report",
                            "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/176997"
                        }
                    ]
                }
            }
        }
    },
    "cveMetadata": {
        "assignerOrgId": "9a959283-ebb5-44b6-b705-dcc2bbced522",
        "assignerShortName": "ibm",
        "cveId": "CVE-2020-4307",
        "datePublished": "2020-06-02T00:00:00",
        "dateUpdated": "2022-01-13T00:00:00",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0",
    "x_ValidationErrors": [
        "$ -- validator = oneOf"
    ]
}